Modules/_asynciomodule.c

```diff
@@ -3053,6 +3147,65 @@ task_step(asyncio_state *state, TaskObj *task, PyObject *exc)
     }
 }
 
+static int
+task_eager_start(asyncio_state *state, TaskObj *task)
+{
+    assert(task != NULL);
```

Comment:
This is fine, just curious why you added it?

Reply:
to make sure we never call this function with task==NULL - it's an implicit assumption in the function body, this makes the assumption explicit